docs/snippets/common/storybook-preview-auto-docs-custom-docs-container-with-custom-page.ts.mdx

```diff
@@ -0,0 +1,42 @@
+```ts
+// .storybook/preview.ts
```

Comment:
```suggestion
// .storybook/preview.tsx
```

Since I'm using jsx syntax, it would be more precise to have it in the file name.
I've copy-pasted the `storybook-preview-auto-docs-custom-docs-container` snippet, so perhaps update even that?